tests/pipelines/kitctl/run.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
I don't  understand your intention of having 2 different folders here like `kitctl` and `kit-cluster` ?   Those two are essentially pipelines that launch kit clusters , of-course one runs upstream test and other doesn't. I feel like template.yaml is a subset of what `upstream-load.yaml` does, right ? Having them in two different folders doesn't make sense to me. wdyt ?

Reply:
IIUC, the `upstream-load.yaml` is calling the `kit-cluster-create` task which has the assumption that the environment cluster is a EKS cluster created by CDK. This task will not work for an environment created by `kitctl`. 

To provide a working example for `kitctl` cluster, we need a pipeline that uses tasks that only deal with native k8s environment. 

For pipeline folders I think we can combine the `kitctl` and `kit-cluster`. But I think we should separate the tasks folders. If we want to have consistency,  then we will keep pipeline and tasks both separated in different folders.